docs/GlobalAPI.md

```diff
@@ -195,6 +195,8 @@ Also under the alias: `fdescribe(name, fn)`
 You can use `describe.only` if you want to run only one describe block:
 
 ```js
+
+/* eslint jest/no-focused-tests: "off" */
```

Comment:
As this is the description of `describe.only` itself, we have to explicitly turn off the `jest/no-focused-tests` rule for this one by adding a comment. If this causes confusion, can we turn off this rule globally?